flink-tests/src/test/java/org/apache/flink/test/checkpointing/EventTimeAllWindowCheckpointingITCase.java

```diff
@@ -133,9 +122,12 @@ public void apply(
 							out.collect(new Tuple4<>(key, window.getStart(), window.getEnd(), new IntType(sum)));
 						}
 					})
-					.addSink(new ValidatingSink(numKeys, numElementsPerKey / windowSize)).setParallelism(1);
+				.addSink(new ValidatingSink<>(
+					new EventTimeWindowCheckpointingITCase.SinkValidatorUpdateFun(numElementsPerKey),
+					new EventTimeWindowCheckpointingITCase.SinkValidatorCheckFun(numKeys, numElementsPerKey, windowSize)))
+				.setParallelism(1);
 
-			tryExecute(env, "Tumbling Window Test");
+			env.execute("Tumbling Window Test");
```

Comment:
Why is this not using `tryExecute()` anymore?

Reply:
Same as above.